bench/binarytrees/binarytrees.dora

```diff
@@ -44,7 +44,8 @@ fun loops(iterations: Int, depth: Int) {
     depth.toString() + "\t check: " + check.toString());
 }
 
-fun createTree(var depth: Int) -> TreeNode {
+fun createTree(dep: Int) -> TreeNode {
+  var depth = dep;
```

Comment:
For all those replacements I would prefer if we would just have: `var depth = depth`, so not renaming the variable.

Reply:
Ah, right, completely forgot we can shadow like this. That's of course much better, will do it!

Reply:
Done!